bin/build-site.sh

```diff
@@ -46,6 +46,14 @@ if [  -z "$TW5_BUILD_OUTPUT" ]; then
     TW5_BUILD_OUTPUT=./output
 fi
 
+# Check if archive should be built depending on environment variable TW5_BUILD_OUTPUT
+# see: https://github.com/Jermolene/TiddlyWiki5/blob/4b56cb42983d4134715eb7fe7b083fdcc04980f0/.github/workflows/ci.yml#L62
+
+if [[ $TW5_BUILD_OUTPUT = "./output" ]]; then
```

Comment:
I don't like the hardcoding of a special parameter value.  I'd prefer to use a separate environment variable TW5_BUILD_ARCHIVE which accepts the relative path where the archive index file should be written, and no archive would be written if the variable were not set, or was blank.